fpr/views.py

```diff
@@ -140,27 +138,27 @@ def formatversion_delete(request, format_slug, slug):
     version = get_object_or_404(fprmodels.FormatVersion, slug=slug, format=format)
     dependent_objects = utils.dependent_objects(version)
     breadcrumbs = [
-        {'text': 'Formats', 'link': reverse('format_list')},
+        {'text': _('Format versions'), 'link': reverse('format_list')},
         {'text': format.description, 'link': reverse('format_detail', args=[format.slug])},
     ]
     if request.method == 'POST':
         if 'disable' in request.POST:
             version.enabled = False
-            messages.info(request, 'Disabled.')
+            messages.info(request, _('Disabled.'))
             for obj in dependent_objects:
                 obj['value'].enabled = False
                 obj['value'].save()
         if 'enable' in request.POST:
             version.enabled = True
-            messages.info(request, 'Enabled.')
+            messages.info(request, _('Enabled.'))
         version.save()
         return redirect('format_detail', format.slug)
     return render(request, 'fpr/disable.html',
         context({
             'breadcrumbs': breadcrumbs,
             'dependent_objects': dependent_objects,
             'form_url': reverse('formatversion_delete', args=[format.slug, version.slug]),
-            'model_name': 'Format Version',
```

Comment:
Using the __unicode__ repr of the model can result in some pretty long and difficult to parse strings, e.g., "Disable File Extension version 0.1 File extension runs Identify by File Extension with .aifc is Audio: Audio Interchange File Format: Audio Interchange File Format (compressed)".

I notice that the Spanish translation makes this easier to parse since it changes "with .aifc" to "con salida .aifc". Maybe the English should be "with output .aifc". 

I don't know exactly what the solution is, but IMO, if these model reprs are gonna be so prominent in the UI, they should be made easier to understand, something like "Disable the rule that uses the tool “File Extension version 0.1”  and runs the command “Identify by File Extension” to identify “Audio: Audio Interchange File Format: Audio Interchange File Format (compressed)” files as “.aifc”."